src/signal_scope/signaldata.cpp

```diff
@@ -185,6 +185,13 @@ QRectF SignalData::computeBounds()
     if (y > maxY) maxY = y;
   }
 
+  double marginMax = 1.1;
+  double marginMin = 1.1;
+  if (maxY < 0.0) marginMax = 0.9;
+  if (minY > 0.0) marginMin = 0.9;
+  maxY *= marginMax;
+  minY *= marginMin;
```

Comment:
can you explain what the above code is doing?  It seems like it is inflating the Y bounds computed by this function?  Why is that required?  It seems like this function should return the true bounds, and the caller can decide whether or not to inflate the bounds if needed.